### ISSUE TITLE
Restoring DNS does not work with --user nobody --group nobody

### ISSUE THREAD

**Issue by erhan-** [masterkorp/openvpn-update-resolv-conf#12]
Just a small hint: You can't use this in combination with downgrading the group and user. When you quit OpenVPN or disconnect the script gets run as nobody, who does not have any permissions to change /etc/resolv.conf.

**Comment by masterkorp** [comment]
Yes, this is a known issue. Maybe I shall add some information about this on the README file.

**Comment by erhan-** [comment]
Yeah I was having this problem for some years and have been manually setting my resolv.conf.

**Comment by piotr-dobrogost** [comment]
> You can't use this in combination with downgrading the group and user.
You can if you use [down-root](https://github.com/OpenVPN/openvpn/blob/master/src/plugins/down-root/README.down-root) OpenVPN plugin

**Comment by masterkorp** [comment]
:+1: @piotr-dobrogost

**Comment by erhan-** [comment]
Thank you for the hint @piotr-dobrogost !
